Decred Wallet/Utils/LocalizedStrings.swift

```diff
@@ -386,4 +382,13 @@ struct LocalizedStrings {
     static let errorMsg = NSLocalizedString("errorMsg", comment: "")
     static let info = NSLocalizedString("info", comment: "")
     static let delete = NSLocalizedString("delete", comment: "")
+
+    /* Other words(used but not on specific views) */
+    static let send = NSLocalizedString("send", comment: "")
+    static let receive = NSLocalizedString("receive", comment: "")
```

Comment:
Send and receive, I believe are used from the navigation menu floating bar. This would make sense to place just below the strings used on the nav bar.

The others are used on the more tab, irrc.

Reply:
resolved in 37e2453